src/schemas/components/SchemaForm.vue

```diff
@@ -35,4 +45,35 @@
       emit('update:values', values)
     },
   })
+
+  const loadingFallback = ref(false)
+  const loading = computed({
+    get() {
+      return props.loading ?? loadingFallback.value
+    },
+    set(value) {
+      loadingFallback.value = value
+      emit('update:loading', value)
+    },
+  })
+
+  async function submit(): Promise<void> {
+    loading.value = true
+
+    try {
+      const { valid, errors: errorsResponse } = await api.schemas.validate(props.schema, values.value)
+
+      errors.value = errorsResponse
```

Comment:
Just double checking cause I didn't look at the API for this - `/validate` doesn't throw an error if the schema is invalid, right? It returns a 200 + some errors?

Reply:
I believe it returns a 403 if the schema itself is invalid.